### ISSUE TITLE
Xiaomi Alioth (POCO F3) Vendor_Boot not boot

### ISSUE THREAD

**Issue by Rofikkernel** [cfig/Android_boot_image_editor#128]
Thanks for your hardwork @cfig , i hope you still work in this project ABIE because it already five years project i see from 2018

I succesfully unpack vendor_boot.img with vbmeta.img following your instruction

Info
ROM : MIUI 14 Android 13
Host : Ubuntu 22.04 jammy
ABIE version = android 13, june 16 build
DTC =  using device tree compiler package by ubuntu, if using dtc aosp (1.6.0) it cannot decode YAML
**Reproduce**

- copy Vendor_boot & vbmeta to project folder
- Gradlew unpack
- gradlew pack ( i did not modify or touch anything to test it)
- Fastboot flash vendor_boot_ab vendor_boot.img.signed
- fastboot flash vbmeta_ab vbmeta.img.signed
- Device stuck at POCO logo (not MIUI logo) and restart about 30s (it didnot throw or detected in adb, so cannot see what happen but i think if have i have serial cable i can see what going on
i think it failed verify in bootloader or something

Note: i still succesfull boot if only repacked vbmeta im flashing, it fail in if flash repacked vendor_boot.img

screenshot Unpack
![image](https://github.com/cfig/Android_boot_image_editor/assets/26862653/e366da69-ab9d-4f77-ac6d-9f73e69956db)

Screenshot Pack
![image](https://github.com/cfig/Android_boot_image_editor/assets/26862653/55934710-f9e7-402c-868a-83365591611a)
![image](https://github.com/cfig/Android_boot_image_editor/assets/26862653/7024649f-51d1-4ef0-bf72-65a7d3660e14)

i see in some log in repack is

 - cfig.Avb - original image vendor_boot.img.signed doesn't have AVB footer
- avb.blob.AuxBlob - encodePubKey(): No key to encode for algorithm NONE
- no pubkey in auxBlob

here My vendor_boot.img and vbmeta,img for investigation
[Alioth.img.zip](https://github.com/cfig/Android_boot_image_editor/files/12269207/Alioth.img.zip)

**Comment by cfig** [comment]
Yeah I still work on this project from time to time, as I am not (yet) laid off and still have a chance to work  ... 
For the issue you meet, maybe you can refer to https://github.com/cfig/Android_boot_image_editor/issues/117
Someone found MI used a strange format parameter to pack its image. Maybe you can have a try with that ?

**Comment by Rofikkernel** [comment]
thank @cfig where that crc64 setting file location

**Comment by Rofikkernel** [comment]
SOLVED!
What cause that error is device-tree-compiler by ubuntu not compatible with miui 14 dtb

- sudo apt purge device-tree-compiler
- git clone latest branch than release version of android boot editor
- ./gradlew unpack
(now unpacked vendor_boot without dtb.dts source)
-./gradlew pack
- flash new packed files and boot

Workarround for want to modify dtb
i am using prebuilt dtc by aosp 
https://android.googlesource.com/platform/prebuilts/misc/+/refs/heads/android11-release/linux-x86/dtc/
use that dtc for decompile  and compile that dtb
